content/testing-yaml/testing.md

```diff
@@ -21,39 +23,71 @@ npm test
 For non-UI tests or unit testing:
 
 ```bash
-./gradlew test
+- name: Test
+  script: ./gradlew test
+  test_report: app/build/test-results/**/*.xml
 ```
 
 UI tests (also known as instrumented tests):
 
 ```bash
-./gradlew connectedAndroidTest
+- name: Launch emulator
+  script: |
+    cd $ANDROID_HOME/tools
+    emulator -avd emulator &
+    adb wait-for-device
+- name: Test
+  script: |
+    set -e
+    ./gradlew connectedAndroidTest
+    adb logcat -d > emulator.log
+  test_report: app/build/outputs/androidTest-results/connected/*.xml
 ```
 
-## Native iOS
+**Tip:**: you can save the emulator log with `adb logcat -d > emulator.log` command.
```

Comment:
There's double colon. 
Also: ...log with the `adb...` command.

Reply:
fixed in e0d9c8e6222cf49670671fc10c0b345d6644767c